src/index.ts

```diff
@@ -142,21 +146,21 @@ export default class ServerlessConventions {
     // Check that the service name is in kebab case
     if (serviceName !== kebabCase(serviceName)) {
       errors.push(
-        `Warning: Service name "${serviceName}" is not kebab case (dash delimited)`
```

Comment:
Should we change the wording from warning to error?

Reply:
(Also this one is missing the extra `x`)

Reply:
Fixed :D